jetson/power_logging/model/generate_model_summaries.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Directory to store results
+RESULT_DIR = "raw_data/prebuilt_models"
```

Comment:
```suggestion
RESULT_DIR="raw_data/prebuilt_models"
```
Spaces are not allowed on assignment in bash